tests/epyccel/modules/functionals.py

```diff
@@ -60,3 +60,39 @@ def functional_for_3d_range():
 def unknown_length_functional(x : 'int[:]'):
     a = [i*3 for i in range(len(x)*2)]
     return len(a), a[0], a[-1]
+
+def functional_with_enumerate():
+    a = [x + 1 for x in range(10)]
+    b = [i * j for i,j in enumerate(a)]
+    return len(b), b[0], b[1], b[2]
```

Comment:
No need to change but just a remark. You realise that you could simply do:
```suggestion
    return b
```
now right?

Reply:
Thank you! I think I wrote this before it was possible